src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -321,3 +321,13 @@ def _default_openai_function(
             "required": required,
         },
     }
+
+
+def get_tqdm_progress_bar_formatter(title: str) -> str:
```

Comment:
can you place this in a common place so generate can use it too? otherwise this is not a function that can be re-used.

Reply:
addressed in [4d616f6](https://github.com/Arize-ai/phoenix/pull/1757/commits/4d616f63024dc8ccb0b6c5a1099dea83fc3eb058)